clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/CreateServerGroupWithArtifactsSpec.java

```diff
@@ -367,4 +367,131 @@ public void createServerGroup_errorIfNoContainersTest() throws IOException, Inte
         String.format("Failed to detect task failure, in %s seconds", TASK_RETRY_SECONDS),
         TASK_RETRY_SECONDS);
   }
+
+  @DisplayName(
+      ".\n===\n"
+          + "Given description w/ task def artifacts, EC2 launch type, and target group mappings, "
+          + "with multiple load balancers and multiple containers successfully submit createServerGroup operation"
```

Comment:
this description (and test name) are getting quite long. I'm ok with shortening this to just "multiple target group mappings" or "multiple load balancers" if you'd prefer.